### ISSUE TITLE
Add emoji support

### ISSUE THREAD

**Issue by larsenv** [WiiDatabase/RiiTag-Next#23]
I think it would be a good idea to have emoji support in order for people to use with RiiTag, specifically twemoji support. This would be good to use with nicknames and such.

**Comment by RuiNtD** [comment]
Any chance we wanna consider an alternative to Twemoji, since the project seems dead since Elon Musk bought Twitter? [OpenMoji](https://openmoji.org/) is a possible alternative.

**Comment by larsenv** [comment]
Good point. I left Twitter. However the Twemoji should still be OK since Discord relies on it. OpenMoji could be nice to make it look different from Discord but the flags on the tag are really Twemoji. Also, maybe we don't need to do any of this since a good chunk of smilies show up on the tag as the same color as the text. I'm not sure where they're loading it but I kind of like it.

**Comment by RuiNtD** [comment]
It's possible those emojis are being provided by the operating system. I don't know what they look like on RiiTag, but I'd have to assume it's the Microsoft default emojis, which have single color versions.

**Comment by larsenv** [comment]
![image](https://github.com/RiiConnect24/RiiTag-Next/assets/4673969/44e5df90-a4a2-4105-b822-4163e4cd3a0e)
